packages/hash/api/src/collab/graphql/queries/page.queries.ts

```diff
@@ -62,3 +65,36 @@ export const getPageQuery = gql`
   }
   ${pageFieldsFragment}
 `;
+
+export const knowledgePageFieldsFragment = gql`
+  fragment KnowledgePageFields on KnowledgePage {
+    archived
+    summary
+    title
+    ownedById
+    entityId
+    entityVersion
+    contents {
+      ...KnowledgeBlockFields
+    }
+    __typename
+  }
+  ${knolwedgeBlockFieldsFragment}
+`;
+
+export const getKnowledgePageQuery = gql`
+  query getKnowledgePage(
+    $ownedById: ID!
+    $entityId: ID!
+    $entityVersion: String
+  ) {
+    knowledgePage(
+      ownedById: $ownedById
+      entityId: $entityId
+      entityVersion: $entityVersion
+    ) {
+      ...KnowledgePageFields
+    }
+  }
+  ${knowledgePageFieldsFragment}
```

Comment:
But this one is right? I would have expected a compilation error or something... confused

Reply:
I have no idea why this didn't error. I renamed, though

Reply:
So it is, I'm blind